web/templates/base.tmpl

```diff
@@ -12,24 +12,66 @@
   {{block "extra" .}}{{end}}
 
   <div class="sm:mx-auto sm:container">
-    <div class="flex flex-row justify-end space-x-4 my-4 pr-4">
+    <div class="flex flex-row justify-end space-x-4 mt-4 sm:my-4">
       {{$user := is_logged_in}}
       {{if $user}}
-      <a href="{{urlTo "news:overview"}}">{{i18n "NewsTitle"}}</a>
-      <a href="{{urlTo "admin:dashboard"}}">{{i18n "NavAdminDashboard"}}</a>
-      <a href="{{urlTo "admin:allow-list:overview"}}">{{i18n "AdminAllowListTitle"}}</a>
-      <div class="inline">
-        ({{$user.Name}})
-        <a href="{{urlTo "auth:fallback:logout"}}">{{i18n "AuthSignOut"}}</a>
-      </div>
+      <span class="flex flex-row items-center divide-x divide-gray-300">
+        <div class="flex flex-row items-center pr-3">
+          <svg class="text-green-500 w-4 h-4 mr-1" viewBox="0 0 24 24">
+            <path fill="currentColor" d="M22,18V22H18V19H15V16H12L9.74,13.74C9.19,13.91 8.61,14 8,14A6,6 0 0,1 2,8A6,6 0 0,1 8,2A6,6 0 0,1 14,8C14,8.61 13.91,9.19 13.74,9.74L22,18M7,5A2,2 0 0,0 5,7A2,2 0 0,0 7,9A2,2 0 0,0 9,7A2,2 0 0,0 7,5Z" />
```

Comment:
Fancy! Is there a list of these somewhere?

Reply:
I took these from materialdesignicons.com/ they have an export option that presents you the HTML code to copy paste.